dashboard/test/controllers/certificate_images_controller_test.rb

```diff
@@ -33,13 +31,20 @@ class CertificateImagesControllerTest < ActionController::TestCase
     assert_includes response.body, 'invalid donor name'
   end
 
-  test 'can show certificate image given bogus course name' do
```

Comment:
I like that you added a test to document *undesired* behavior and then revised the test once the functionality was as desired